src/types/base.ts

```diff
@@ -85,17 +91,14 @@ export interface Metric {
     | 'restore';
 }
 
-/**
- * A version of the `Metric` that is used with the attribution build.
- */
-export interface MetricWithAttribution extends Metric {
```

Comment:
this isn't used or referred to from anywhere—metrics with attribution extend their individual metrics directly (e.g. `LCPMetricWithAttribution extends LCPMetric`)—so I removed, but I can restore if it's still useful for something

Reply:
I'm not exactly sure if I did this on purpose and in this case, but in many cases I created a type that I could just copy and paste to the README for the docs. E.g. https://github.com/GoogleChrome/web-vitals/#metricwithattribution

Not the most sophisticated doc generation :) but the point was that the docs would be easy to create from the types files.

In this case if it's truly not being used then I guess there's no additional "checking" that's happening by keeping it here, so it's probably fine to remove. But then I guess the question is what should we put in the docs?

Reply:
probably easier to revert and come at it if/when the documentation changes :)

Reply:
I added this in #356 so definitely good to remove this so you're cleaning up #356 in this. And, as it's a PR into that branch, we should merge this and then merge #356

Reply:
#348 added `ReportCallbackWithAttribution`, `MetricWithAttribution` was added in #237

> And, as it's a PR into that branch, we should merge this and then merge #356

Oh, I was only doing that so the diff made sense. Sorry to hold you up. You can merge #356 any time and I'll just change the PR to compare against `main`.

Reply:
Whichever's easier for you!